scalalib/overrides-2.12/scala/App.scala

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Are the overrides just standard for `2.12` even in the Scala repo?

Reply:
Hmm, what do you mean?

Reply:
I am just asking a general question about why these are here. They are not changed for Scala Native are they?

Reply:
Oh yeah, we leave them as is. Basically, where applicable, I applied the changelog from the `overrides-2.11` to the 2.12 files.